conf/snapshot_dir_conf.py

```diff
@@ -0,0 +1,6 @@
+"""
+Conf file for snapshot directory
+"""
+import os
+
+snapshot_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', 'tests', 'snapshots', 'test_accessibility', 'test_accessibility', 'chrome')
```

Comment:
Why do we have `test_accesibility` twice?
Do we intend to store the snapshots in `/root/project/tests/snapshots/test_accessibility/test_accessibility/chrome/` dir?

Reply:
That's Right @shivahari .